framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java

```diff
@@ -241,6 +246,54 @@ private void executeQuery(String query) throws IOException, SQLException {
     }
   }
 
+  private void executeLimitZeroQuery(String query) throws IOException, SQLException {
+	if (getTestStatus() == TestStatus.CANCELED) {
+		return;
+	}
+	
+	BufferedWriter writer = new BufferedWriter(new FileWriter(new File(outputFilename),true));
+	
+    Statement statement = connection.createStatement();
+    ResultSet resultSet = statement.executeQuery(query);
+    
+    List columnLabels = new ArrayList<String>();
+    List<Integer> types = Lists.newArrayList();
+    List<Integer> nullability = Lists.newArrayList();
+    try {
+      int columnCount = resultSet.getMetaData().getColumnCount();
+      for (int i = 1; i <= columnCount; i++) {
+        columnLabels.add(resultSet.getMetaData().getColumnLabel(i));
+        types.add(resultSet.getMetaData().getColumnType(i));
+        nullability.add(resultSet.getMetaData().isNullable(i));
+      }
+      
+      String msg = "\n" + query + "\n" + columnLabels.toString() + "\n"
+    		  + this.columnLabels.toString() + "\n"
+    		  + "\n" + Utils.getTypesInStrings(types).toString() + "\n"
+    		  + Utils.getTypesInStrings(this.types).toString() + "\n"
+    		  + "\n" + nullability.toString() + "\n"
```

Comment:
Please add `Utils.getNullabilityStatusesInStrings(...)` for better readability of statuses. The status names are listed in the `java.sql.ResultSetMetaData` class.